lib/model/Joi.js

```diff
@@ -14,6 +14,13 @@ const EXTENDED_JOI = Joi.extend({
     if (value === undefined) {
       return { value };
     }
+    const { _valids: valids } = helpers.schema;
```

Comment:
is there really no built-in way to run any chained `.allow()` or `.optional()` validators?  we shouldn't have to re-implement all these in our custom types.

Reply:
(recommending that we explore that later in a separate PR)